docs/overview/style-props.md

```diff
@@ -3,901 +3,743 @@
 ## Size and position
 Properties related to size, position, alignment and layout of the objects.
 
-
 ### width
 Sets the width of object. Pixel, percentage and `LV_SIZE_CONTENT` values can be used. Percentage values are relative to the width of the parent's content area.
 <ul>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Default</strong> widget dependent</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Inherited</strong> No</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Layout</strong> Yes</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Ext. draw</strong> No</li>
+<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Default</strong>Widget dependent</li>
```

Comment:
There should be a space after `</strong>`:
```html
<strong>Default</strong> Widget dependent
```

Reply:
It's strange that: there is one space before "Widget dependent", but no space after it. So, I change the format.

Reply:
The space after the items is created with `margin-right:20px` of the `<li>` elements.

Reply:
Hmm, understand. Fixed, @kisvegabor